geode-core/src/main/java/org/apache/geode/admin/jmx/internal/RefreshNotificationType.java

```diff
@@ -32,25 +31,21 @@
   /** Notify StatisticResource to refresh statistics */
   @Immutable
   public static final RefreshNotificationType STATISTIC_RESOURCE_STATISTICS =
-      new RefreshNotificationType("GemFire.Timer.StatisticResource.statistics.refresh", "refresh");
+      new RefreshNotificationType("GemFire.Timer.StatisticResource.statistics.refresh", "refresh",
```

Comment:
Minor - can "refresh" be pulled into a final shared field? for this line and the one below it?